src/EPR.RegulatorService.Facade.Core/Services/Regulator/RegulatorOrganisationService.cs

```diff
@@ -132,10 +142,16 @@ private StringContent GetStringContent(object request)
         public async Task<HttpResponseMessage> GetUsersByOrganisationExternalId(Guid userId, Guid externalId)
         {
             var url = string.Format($"{_config.Endpoints.GetUsersByOrganisationExternalId}", userId, externalId);
-        
-            _logger.LogInformation("Attempting to fetch the users for organisation external id {externalId} from the backend", externalId);
-        
-            return await _httpClient.GetAsync(url);
+
+            string logData = string.Format($"Attempting to fetch the users for organisation external id {0} from the backend", externalId);
+            _logger.LogInformation("{Message}", logData);
+
+            var uriBuilder = new UriBuilder(_httpClient.BaseAddress)
```

Comment:
Please undo this change, ignore the SQ warning as peter is happy to leave it as is